spylon_kernel/scala_interpreter.py

```diff
@@ -499,12 +499,18 @@ def bind(self, name: str, value: Any, jtyp: str="Any"):
 
     @property
     def jcompleter(self):
+        """Scala code completer.
+
+        Returns
+        -------
+        scala.tools.nsc.interpreter.PresentationCompilerCompleter
```

Comment:
this looks almost fully qualified. Should this be: `py4j.java_gateway.JVMView. scala.tools.nsc.interpreter.PresentationCompilerCompleter` ?